src/k8s-extension/azext_k8s_extension/partner_extensions/OpenServiceMesh.py

```diff
@@ -93,3 +105,58 @@ def Update(self, extension, auto_upgrade_minor_version, release_train, version):
             release_train=release_train,
             version=version
         )
+
+
+def _validate_tested_distro(cmd, cluster_resource_group_name, cluster_name, extension_version):
+
+    if version.parse(str(extension_version)) <= version.parse("0.8.3"):
+        logger.warning('\"testedDistros\" field unavailable for version %s of osm-arc, '

```

Comment:
Can this please be changed to `"testedDistros" field is unavailable for version %s of microsoft.openservicemesh. Unable to determine if this Kubernetes distro has been properly tested.`